### PR TITLE
Fixes #77

### LINKED ISSUE #77: Impropper pins cause constant service reset
```
Mar  3 07:49:13 controller01 puppet-agent[53523]: (/Stage[main]/Mesos::Install/Package[mesos]/ensure) ensure changed '0.27.1-2.0.226.ubuntu1404' to '0.27.1*'
Mar  3 07:49:13 controller01 puppet-agent[53523]: (/Stage[main]/Marathon::Install/Package[marathon]/ensure) ensure changed '0.15.3-1.0.463.ubuntu1404' to '0.15.3*'
Mar  3 07:49:14 controller01 puppet-agent[53523]: (/Stage[main]/Consular/Package[python-consular]/ensure) ensure changed '1.2.0-1ppa1' to '1.2.0*'
Mar  3 07:49:14 controller01 kernel: [287551.487746] init: mesos-master main process (53376) killed by TERM signal
Mar  3 07:49:14 controller01 puppet-agent[53523]: (/Stage[main]/Mesos::Master/Mesos::Service[master]/Service[mesos-master]) Triggered 'refresh' from 1 events
```

### PR DESCRIPTION
**Pull request by calston**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "praekeltfoundation-seed_stack",
-  "version": "0.6.2-dev",
+  "version": "0.6.3-dev",
```
Comment (JayH5) [inline metadata.json]: The version is "pre-bumped". i.e. 0.6.1 is the current latest version. We remove the `-dev` before publishing a new version to puppet forge.

File: spec/classes/controller_spec.rb [hunk 2]
```diff
@@ -59,7 +59,7 @@
         end
         it do
           is_expected.to contain_class('mesos')
-            .with_ensure(/\d+\.\d+\.\d+\*/)
+            .with_ensure(/\d+\.\d+\.\d+.+$/)
```
Comment (JayH5) [inline spec/classes/controller_spec.rb]: can you add a leading `^` to these please?

**Comment by jerith** [comment]
The README should probably be updated to note that we're using OS-specific packages.

**Comment by JayH5** [comment]
:+1:

I think it's pretty clear that we only support Ubuntu 14.04 currently. Not too worried about the README. I can add a small note in the changelog when I do the release.

**Comment by calston** [comment]
Is there a mechanism to override the params?

**Comment by JayH5** [comment]
> Is there a mechanism to override the params?

On a class-by-class basis yes. And soon in #74 all in one place.

**Comment by calston** [comment]
Basically everything is entirely untested on deadrat/centos/fedora/slackware/whatever so that's a giant gardenpath anyway. This is our seed_stack so I don't feel the need to go down it.

**Comment by calston** [comment]
Branch tested on Nigeria stack, confirmed fixes issue.

**Pull request merged**